.github/workflows/build-test.yml

```
@@ -26,7 +26,7 @@ on:
     - cron: '0 9 * * 1-5' # 9am UTC, Monday-Friday (2am PDT, after VS Code Insider builds which is 11pm PDT)
   workflow_dispatch:
 env:
-  NODE_VERSION: 12.14.1
+  NODE_VERSION: 14.17.5
```

I'd change this to 14.16.0, the same version as used by VS Code.
```suggestion
  NODE_VERSION: 14.16.0
```

---

@DonJayamanne I was just trying to look this up (I recalled that VSCode had a specific version) but couldn't find it off hand. Where is this? Wasn't seeing it in yarn or package files. Just so I can look it up myself.